modules/acquia_cms_search/src/Plugin/AcquiaCmsTour/AcquiaConnectorForm.php

```diff
@@ -60,15 +91,24 @@ public function buildForm(array $form, FormStateInterface $form_state) {
         '#collapsible' => TRUE,
         '#collapsed' => TRUE,
       ];
+      // Start with an empty subscription.
+      $subscription = $this->subscription->getSubscription(TRUE);
+      $site_name = isset($subscription['uuid']) ? $this->siteProfile->getSiteName($subscription['uuid']) : "";
+      $form[$module]['opt_in'] = [
+        '#type' => 'checkbox',
+        '#title' => $this->t('Send anonymous data about Acquia product usage'),
+        '#default_value' => $this->state->get('acquia_connector.telemetry.opted') ?? $this->isModuleEnabled(),
+        '#description' => $this->t('This module intends to collect anonymous data about Acquia product usage. No private information will be gathered. Data will not be used for marketing or sold to any third party. This is an opt-in module and can be disabled at any time by uninstalling the acquia connector module by your site administrator.'),
+        '#prefix' => '<div class= "dashboard-fields-wrapper">' . $module_info['description'],
+      ];
       $form[$module]['site_name'] = [
         '#type' => 'textfield',
         '#title' => $this->t('Name'),
         '#maxlength' => 255,
         '#disabled' => TRUE,
         '#required' => TRUE,
-        '#default_value' => $this->state->get('spi.site_name'),
-        '#prefix' => '<div class= "dashboard-fields-wrapper">' . $module_info['description'],
-        '#suffix' => "</div>",
+        '#default_value' => $this->state->get('spi.site_name') ?? $site_name,
+        '#suffix' => '</div class= "dashboard-fields-wrapper">',
```

Comment:
any reason for removal of prefix here?

Reply:
Yes to make UI like this.
#prefix attribute is moved to checkbox field.
<img width="1672" alt="Screenshot 2022-11-04 at 6 41 25 PM" src="https://user-images.githubusercontent.com/106175376/199980467-32a6c4be-a65f-4f83-8119-b0ca1ff6111c.png">